app/viewmodels/tasklist/VatRegistrationTaskList.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.tasklist
+
+import config.FrontendAppConfig
+import models.api.{NETP, VatScheme}
+import models.api.vatapplication.{OverseasCompliance, StoringWithinUk}
+import models.{ConditionalValue, CurrentProfile, NonUk}
+import play.api.i18n.Messages
+import play.api.mvc.Request
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class VatRegistrationTaskList @Inject()(aboutTheBusinessTaskList: AboutTheBusinessTaskList) {
+
+  def buildGoodsAndServicesRow(implicit profile: CurrentProfile): TaskListRowBuilder = TaskListRowBuilder(
+    messageKey = _ => "tasklist.vatRegistration.goodsAndServices",
+    url = scheme => {
+      scheme.partyType match {
+        case Some(NETP) => controllers.vatapplication.routes.TurnoverEstimateController.show.url
+        case _ => controllers.vatapplication.routes.ImportsOrExportsController.show.url
+      }
+    },
+    tagId = "goodsAndServicesRow",
+    checks = scheme => {
+      checkImportsAndExports(scheme).++ {
+        Seq(
+          scheme.vatApplication.exists(_.turnoverEstimate.isDefined),
+          scheme.vatApplication.exists(_.zeroRatedSupplies.isDefined),
+          scheme.vatApplication.exists(_.northernIrelandProtocol.exists(_.goodsToEU.exists(checkConditionalValue))),
```

Comment:
You could just assert it to be isDefined here as the boolean is always there if the ConditionalValue is with how it's set up.